Regular-Question-Answers/next-permutation.cpp

```diff
@@ -0,0 +1,38 @@
+//Author: Toygun Görmüş
+//Problem Link : https://leetcode.com/problems/next-permutation/
+/*Using two pointers will be efficient in this problem. The two pointers will point to the first not sorted element and the first greater element after it. Thus, by swapping these numbers and sorting the rest of the array from the first not sorted element should yield the next permutation.  */

```

Comment:
descending diye bir fonksiyon kullanmak yerine ilk while'ı bunun kontrolünde de kullanabilirsen daha iyi olur. 
26 28 31 ve 27 29 32 yi alt alta tutman whileların ne yaptığını görmek için daha yaralı olur.
Parantezler yerli yerinde değiller. 
ve yorum satırın çok uzun olduğu için sayfada tamamen görünmüyor, o yalnız buraya özgü mü, yüklenince sıkıntı olmuyor mu bilmiyorum ama o kısmı düzeltmek isteyebilirsin